src/utils/validation.ts

```diff
@@ -28,11 +30,32 @@ const validateReferralUsername = (username: string) => {
   return re.test(String(username));
 };
 
+const validateSignUpForm = ({
+  email,
+  password,
+  confirmPassword,
+  phone,
+  username,
+  referralUsername,
+  isTermsAgree,
+  isPrivacyAgree,
+} : FormData) => ({
+  email: validateEmail(email),
+  password: validatePassword(password),
+  confirmPassword: validateConfirmPassword(password, confirmPassword),
+  phone: validatePhone(phone),
+  username: validateUsername(username),
+  referralUsername: referralUsername === '' ? true : validateReferralUsername(referralUsername),
+  isTermsAgree: isTermsAgree === true,
```

Comment:
+ isTermsAgree가 boolean이면 '=== true'를 생략해도 될 것 같습니다.
```suggestion
  isTermsAgree: isTermsAgree
```

+ 그리고 `something === true`를 사용하는 곳이 많은데 표현을 함수로 바꾸어 보는 것은 어떨까요?
```js
const isTrue = (arg) =>  arg === true;
```